@xen-orchestra/web/src/components/site/SiteHeader.vue

```diff
@@ -19,7 +19,7 @@
       <TabItem :active="isExactActive" :href tag="a">{{ t('pools') }}</TabItem>
     </RouterLink>
     <TabItem disabled>{{ t('hosts') }}</TabItem>
-    <TabItem disabled>{{ t('vms') }}</TabItem>
+    <TabItem disabled>{{ t('vms', 0) }}</TabItem>
```

Comment:
According to Figma, show the vms variant.
```suggestion
      {{ t('vms', 2) }}
```